rep-2005.rst

```diff
@@ -39,20 +39,53 @@ The list below is curated to include repositories that are:
 
 We aim that all of these repositories are also:
 
-* actively maintained and following accepted software development processes, including code review and testing.
+* actively maintained and following accepted software development processes, including code review and testing; and
+* at Quality Level 3 or better, as defined in REP 2004.
```

Comment:
Quality level 3 seems reasonable, but didn't we discuss this in the context of the guidelines (also mentioned below)? It seems like the quality reference here should be removed and we should instead augment L76 as: "meet a reasonable quality standard as defined in REP 2004, ideally Quality Level 3 or higher".

This makes it clear that it is guidance and not a strict requirement. This comment itself doesn't create a strict requirement because of the wording "We aim..." above - so compiling all the guidance in one section helps keep the rest of the REP more strict.

Reply:
I take your point. But I see the rules and guidance falling into 3 buckets:

1. Requirements that are already in effect. These are the contents of the first list under `Scope`.
1. Requirements that we wish were in effect, but that we can't enforce yet because it would be impractical. These are the contents of the second "We aim..." list under `Scope`.
1. Qualitative guidance that should be used when trying to decide on a proposal that meets at least those requirements that are already in effect. These are the contents of the list under `Guidance`.

So the second list under `Scope` is a kind of incubation area for rules that may one day be enforced. Those rules should be reasonably crisp and enforceable so that they can be promoted later. By contrast, the items listed under `Guidance` are intended to remain there in the long run as advice on how to make judgment calls for cases where the rules are already met.

To better to match this structure, I made a couple of tweaks in 
320f53c and 563dfe3.

Reply:
+1 to @gerkey's answer.